docs/VERSIONING.md

```diff
@@ -3,11 +3,13 @@
 DiracX is a comprehensive Python package, composed of several interconnected submodules. It's designed to provide robust and versatile functionalities, primarily through these key components:
 
 1. **User-Facing Components**:
+
    - **`diracx`**: This is the primary interface for users, integrating both the Command Line Interface (CLI) and Python API.
    - **`diracx-routers`**: Serves as the server component, offering HTTPS endpoints.
    - **`diracx-tasks`**: Handles operations executed by DiracX servers, either periodically or upon request.
 
-2. **Containerization**:
+1. **Containerization**:
```

Comment:
wrong

Reply:
I noticed that. This is dependant, not me ...